source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -65,13 +65,13 @@ namespace PlayFab
         )
         {
             IPlayFabHttpPlugin& http = *PlayFabPluginManager::GetPlugin<IPlayFabHttpPlugin>(PlayFabPluginContract::PlayFab_Transport);
-            const auto requestJson = request.ToJson();
+            const auto& requestJson = request.ToJson();
```

Comment:
>auto [](start = 18, length = 4)

ToJson doesn't clearly state what the type is. recommend expanding to use the full type here